scripts/soft-reboot

```diff
@@ -93,7 +93,7 @@ function clear_lingering_reboot_config()
     if [[ -f ${WARM_DIR}/${REDIS_FILE} ]]; then
         mv -f ${WARM_DIR}/${REDIS_FILE} ${WARM_DIR}/${REDIS_FILE}.${TIMESTAMP} || /bin/true
     fi
-    /sbin/kexec -u || /bin/true
+    /sbin/kexec -u -a || /bin/true
```

Comment:
Why is this change required?

Reply:
I have the same question. If this is valid and general bug fix, we need to evaluate the backport needs. Could you separate it into a standalone PR?

Reply:
kexec -u on systems with SB enabled is failing. Already fixed for WR here: https://github.com/sonic-net/sonic-utilities/pull/3439. 

Since we are updating soft-reboot to work on Secure boot systems. i've also added this change